exporter/prometheusremotewriteexporter/wal_test.go

```diff
@@ -101,6 +103,7 @@ func TestWAL_persist(t *testing.T) {
 
 	pwal, err := newWAL(config, doNothingExportSink)
 	require.Nil(t, err)
+	pwal.log = zap.Must(zap.NewDevelopment())
```

Comment:
To make your life easier, I suggest using `zaptest.NewLogger(t)` within tests so that you get logs only if the tests fails.